### PR TITLE
chore: move main.go from tracee-ebpf & tracee-rules

### PR DESCRIPTION
**Pull request by rafaeldtinoco**
1. I'm moving tracee-ebpf/3rdparty to ./3rdparty. It seems more natural to me to have a 3rdparty directory inside the root directory (containing 3rdparty software/code to all our project).

2. I included commit from @danielpacak ("chore: move tracee-rules/main.go to cmd/tracee-rules/main.go") just so we could have a unified view of the change in the same PR.

3. I don't think printer.go belong to ./cmd/tracee-ebpf/ like it is being suggested. But I **did not** want to create a pkg for it, just yet. So we can move it once @mtcherni95 decides what to do with printer.go (after this PR moving tracee-ebpf/main.go to the correct place).

### CODE REVIEW

File: Makefile.one [hunk 1]
```diff
@@ -45,6 +47,21 @@ checklib_%:
...
+OS_VERSION_ID := $(shell $(CMD_GREP) -m1 'VERSION_ID=' $(OS_RELEASE_FILE) | $(CMD_CUT) -d= -f2 | $(CMD_SED) 's/\"//g')
+
+ifneq (,$(findstring rhel,$(OS_ID_LIKE)))
+        RHEL_MAJOR := $(shell echo $(OS_VERSION_ID) | cut -d. -f1)
+        RHEL_MINOR := $(shell echo $(OS_VERSION_ID) | cut -d. -f2)
+        RHEL_RELEASE_CODE := $(shell echo $$(($(RHEL_MAJOR) << 8 | $(RHEL_MINOR))))
+else
+        RHEL_RELEASE_CODE := 0
+endif
+
```
Comment (yanivagman) [inline Makefile.one:64]: Why do we need this?
CO-RE doesn't require this field (it is now guarded by #ifndef CORE in bpf code), and if the bpf object is compiled with RHEL headers, it already exists.
Not only this, I think we can totally revert #896 (and modifications made later related to RHEL and CORE).

File: builder/Dockerfile.mkdocs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+FROM squidfunk/mkdocs-material:8.0.1
+
+RUN pip install --upgrade pip && \
+    pip install mike && \
+    pip install mkdocs-macros-plugin && \
+    pip install git+https://github.com/itaysk/mkdocs-macros-gittagversion
```
[the remark this answers is not in the document]
Comment (danielpacak) [reply builder/Dockerfile.mkdocs:6]: Do we still need such custom macro with top-level Go module? I think we could eventually use built-in `{{ git.tag }}` now.

**Review by danielpacak — APPROVED** [review]
LGTM!

**Comment by itaysk** [comment]
> Do we still need such custom macro with top-level Go module? I think we could eventually use built-in {{ git.tag }} now.

good point! However if it's keeping this PR from merging let's fix in the next since this PR is supposed to not change the code (@yanivagman also related to your comments)

File: Makefile.one (hunk 1 above)
Comment (rafaeldtinoco) [reply Makefile.one:64]: I haven't realized we had it protected by the ifdef CORE changes, indeed. Dropping this commit...

File: builder/Dockerfile.mkdocs (hunk 2 above)
[the remark this answers is not in the document]
Comment (yanivagman) [reply builder/Dockerfile.mkdocs:6]: We can make this change in a future PR.
@danielpacak do you mind to open an issue for your suggestion?

**Review by yanivagman — APPROVED** [review]

File: builder/Dockerfile.mkdocs (hunk 2 above)
[the remark this answers is not in the document]
Comment (danielpacak) [reply builder/Dockerfile.mkdocs:6]: Yup, we have #1246 to follow up on that

**Pull request merged**